.yarn/versions/e97b6ff3.yml

```diff
@@ -0,0 +1,5 @@
+releases:
+  "@radix-ui/react-navigation-menu": patch
```

Comment:
I guess this is technically breaking? not sure how much it matters?

Reply:
Yeah, though I would think probably rare, and we can always call it out in the changelog.